StaticSiteGenerator.UnitTests/Doubles/FileManipulation/FileSytemAutoMockerExtensions.cs

```diff
@@ -0,0 +1,31 @@
+using System.IO.Abstractions;
+using System.IO.Abstractions.TestingHelpers;
+using Moq.AutoMock;
+
+namespace StaticSiteGenerator.UnitTests.Doubles.FileManipulation;
+
+public static class FilesystemAutoMockerExtensions
+{
+    public static void MockFileSystem(this AutoMocker mocker, string[] filePaths)
```

Comment:
This interface is probably not good enough for reuse.  I should probably just stick to one of the internal types in the file system mocker.